pkgs/applications/misc/bb/default.nix

```diff
@@ -18,10 +18,13 @@ stdenv.mkDerivation rec {
     sed -i -e '/^#include <malloc.h>$/d' *.c
   '';
 
+  NIX_CFLAGS_COMPILE = lib.optionalString (stdenv.isDarwin && stdenv.isAarch64)
+    "-D__STRICT_ANSI__";
```

Comment:
Why does this fix the build? Would be nice to have a little comment here.

Reply:
Added comment.